packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -43,6 +44,9 @@ const useStyles = makeStyles(
       fontWeight: theme.typography.fontWeightBold,
       color: theme.palette.textContrast,
     },
+    overlaySmall: {
+      fontSize: theme.typography.pxToRem(25),
```

Comment:
is there a reason for `25px`? alternatively you could go with `theme.typography.h4` (or `h5` or `body`) to have it adjust the theme https://mui.com/material-ui/customization/typography/#responsive-font-sizes

Reply:
The 25px reflects the font inside the badge (the percentage), which in the normal sized version is 45px (a few lines above, right above what's capped here in the preview). I guess this number was chosen because it's big, yet fits inside the circle. 25px fits inside the smaller circle. This is what's used for the `<Gauge>` component (the actual circle thingie).

Reply:
Alirght - the `45px` is also kind of arbitrary & not themable... I'm ok with the `25px`.